typescript/site/app/ecosystem/partners-data/slam-ai/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "SLAMai",
+  "description": "Fast, accurate, agentic data. SLAMai is the definitive platform for smart money intelligence — live on Base and Ethereum. It delivers best-in-class data, open APIs for builders, and an MCP layer purpose-built for autonomous agents.",
+  "logoUrl": "/logos/slam-ai.png",
+  "websiteUrl": "https://www.slamai.xyz/",
+  "category": "Infrastructure & Tooling"
```

Looks good @brunocasarotti but I feel the `Services/Endpoints` category would be more appropriate, see
https://github.com/coinbase/x402/tree/main/typescript/site#servicesendpoints
```suggestion
  "category": "Services/Endpoints"
```